README.md

```diff
@@ -25,6 +25,71 @@ swift-installer-scripts
            └ ...
 ~~~
 
+## Linux Packages (RPM/Deb)
+
+Currently Swift on Linux is distributed via tarball and Docker, and
+we would like to start supporting RPM and Debs officially on swift.org.
+The goal is to provide a seamless install process for Swift on Linux by
+utilizing the platform’s native package manager (RPM/Deb).
+
+
+* Step 1. Develop native packages / installers for the distributions
+* Step 2. Offer the native packages / installers through swift.org
+  * Support all officially supported Linux platforms
+  * Code signed by swift.org certificate
+  * Repository hosted on swift.org
+* Step 3. Offer the native packages / installer through official repositories
+for the various platforms
+  * Work with official repositories to accept package specs
+  * Deprecate swift.org packages / installer repository
+* Step 4. Deprecate swift.org Linux tarballs
+
+### Package Info
```

Comment:
@futurejones @tachoknight @compnerd please review and see if you have any feedback on the suggested metadata here (this is a draft)

Reply:
I think that we should consider the renaming policy for Debian: https://wiki.debian.org/RenamingPackages.  It may still be possible to use swift as the package name.  While it is common for some languages (e.g. go) to use the lang suffix, it generally goes by golang and rather than go.  Rust does something similar: rust is the package name and provides an uber package (rust-all) for installing all the pieces together.